charts/mlflow/README.md

```diff
@@ -19,36 +21,64 @@ Different steps are necessary to deploy this Helm Chart to Rahti2:
 
    You can also use another external S3 service instead of Allas.
 
-2. Deploy MLflow:
+2. By default, it also uses our CSC database service named [Pukki](https://pukki.dbaas.csc.fi). Be sure to have a database created on this service.  
+During the process of creation of database, it will ask you the `Allowed CIDRs`. Rahti2 has a common egress IP which is `86.50.229.150`. If you want a dedicated egress IP, you can send a ticket to [servicedesk@csc.fi](mailto:servicedesk@csc.fi). More information [here](https://docs.csc.fi/cloud/rahti2/networking/#egress-ips).
+
+   A database named `mlflow_auth` must be created when launching your instance. This database is needed for the auth module (only if `tracking.auth.enabled=true` which is the case by default).
+
+3. Deploy MLflow:
 
      ```sh
-     helm install mlflow . --set externalS3.accessKeyID={ACCESS_KEY} --set externalS3.accessKeySecret={SECRET_KEY} --set externalS3.bucket=mlflow
+     helm install mlflow . --set externalS3.accessKeyID={ACCESS_KEY} --set externalS3.accessKeySecret={SECRET_KEY} --set externalS3.bucket={BUCKET_NAME} --set externalDatabase.host={DB_PUBLIC_IP} --set externalDatabase.user={DB_USER} --set externalDatabase.password={DB_PASSWORD} --set externalDatabase.database={DB_NAME}
```

Comment:
Can you cut the command in several lines? IMHO it makes it much more readable in documentation. Something like this:

```suggestion
     helm install mlflow . --set externalS3.accessKeyID={ACCESS_KEY} \
     --set externalS3.accessKeySecret={SECRET_KEY} \
     --set externalS3.bucket={BUCKET_NAME} \
     --set externalDatabase.host={DB_PUBLIC_IP} \
     --set externalDatabase.user={DB_USER} \
     --set externalDatabase.password={DB_PASSWORD} \
     --set externalDatabase.database={DB_NAME}
```